connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/internal/UplynkEventDispatcher.kt

```diff
@@ -28,6 +31,26 @@ internal class UplynkEventDispatcher {
         listeners.forEach { it.onPreplayFailure(e) }
     }
 
+    fun dispatchAdBreaksUpdatedEvents(ads: UplynkAds) {
```

Comment:
I believe we should use `handler.post` for these?
```suggestion
    fun dispatchAdBreaksUpdatedEvents(ads: UplynkAds) = handler.post {
```
Same for the methods below.

Reply:
Good catch!

Reply:
Done